<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>programGroupBox</class>
 <widget class="QGroupBox" name="programGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string/>
  </property>
  <property name="alignment">
   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="plikGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Plik</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="openAviButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Otw√≥rz z dysku</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nazwa_pliku">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Brak pliku</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="convertButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Konwertuj video</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="wideCheckBox">
              <property name="text">
               <string>16:9</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QLabel" name="MP4label">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>MP4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="mp4ProgressBar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QLabel" name="OGVlabel">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>OGV</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="ogvProgressBar">
              <property name="value">
               <number>0</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QLabel" name="WEBMlabel">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>WEBM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="webmProgressBar">
              <property name="value">
               <number>0</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="console">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QGroupBox" name="videoGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>369</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>999</width>
         <height>369</height>
        </size>
       </property>
       <property name="title">
        <string>Odtwarzacz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="videoPlayer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>226</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QPushButton" name="startButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Play</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="textedit.qrc">
                  <normaloff>:/images/media-playback-start.png</normaloff>:/images/media-playback-start.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pauseButton">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Pause</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="textedit.qrc">
                  <normaloff>:/images/media-playback-pause.png</normaloff>:/images/media-playback-pause.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="totalTimeMin">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Czas ca≈Çkowity [min]</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::WinPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="smallDecimalPoint">
                 <bool>false</bool>
                </property>
                <property name="digitCount">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="totalTimeSec">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Czas ca≈Çkowity [sec]</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::WinPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="digitCount">
                 <number>2</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stopklatkaButton">
                <property name="toolTip">
                 <string>Zapisz stopklatkƒô</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="textedit.qrc">
                  <normaloff>:/images/insert-image.png</normaloff>:/images/insert-image.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QGroupBox" name="daneGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>280</height>
        </size>
       </property>
       <property name="title">
        <string>Dane do bazy</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="tytulLabel">
          <property name="text">
           <string>Tytu≈Ç</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tytulLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Podaj tytu≈Ç programu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Opis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="opis">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QToolButton" name="charTableButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="textedit.qrc">
              <normaloff>:/images/character-set.png</normaloff>:/images/character-set.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="cykleLabel">
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Z cyklu:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cykleCombo">
            <property name="toolTip">
             <string>Wybierz lub wpisz tytu≈Ç nowego cyklu</string>
            </property>
            <property name="accessibleName">
             <string>Wybierz lub wpisz tytu≈Ç nowego cyklu</string>
            </property>
            <property name="accessibleDescription">
             <string>Wybierz lub wpisz tytu≈Ç nowego cyklu</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="katButton">
            <property name="text">
             <string>KATEGORIE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="stopklatkaBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>92</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>92</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>90</height>
           </size>
          </property>
          <property name="title">
           <string>Stopklatka</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="stopKlatkaLabel">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>72</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>72</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>72</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="textedit.qrc"/>
 </resources>
 <connections/>
</ui>
